src/components/ModalDetailsMovie/ModalDetailsMovie.js

```diff
@@ -0,0 +1,215 @@
+import React, { Component } from "react";
+import AppContext from "../../app-context";
+import styles from "./ModalDetailsMovie.module.scss";
+import SmallButtonActionWatchlist from "../SmallButtonActionWatchlist/SmallButtonActionWatchlist";
+import IconClose from "../../assets/icons/close.svg";
+import RatingIcon from "../../assets/icons/rating.svg";
+import defaultPoster from "../../assets/single-logo.png";
+import BigButtonActionWatchlist from "../BigButtonActionWatchlist/BigButtonActionWatchlist";
+import Loading from "../Loading/Loading";
+
+class ModalDetailsMovie extends Component {
+  state = {
+    isLoading: true,
```

Comment:
to powinno być z defaultu na false,
gdy zaczyna się fetch - powinno zmieniać się na true,
gdy kończy się akcja z fetcha - spowrotem na false